CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
+documentation, we greatly value feedback and contributions from our community.
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
+information to effectively respond to your bug report or contribution.
+
+
+## Reporting Bugs/Feature Requests
+
+We welcome you to use the GitHub issue tracker to report bugs or suggest features.
```

Comment:
We need to differentiate between what is entered in the git trackers, and what is directed to https://forums.freertos.org

Reply:
I will rephrase this to ask people to post and discuss on the forum first to avoid disappointment as per the next comment as well.